.netlify/_redirects

```diff
@@ -31,4 +31,15 @@
 /newsletter	                          https://substrate.io/ecosystem/connect/newsletter/             301!
 /seminar	                          https://substrate.io/ecosystem/resources/seminar/              301!
 /terms	                              https://www.parity.io/terms/                                   301!
-/awesome-substrate                    https://github.com/substrate-developer-hub/awesome-substrate/blob/master/README.md    301!
\ No newline at end of file
+/awesome-substrate                    https://github.com/substrate-developer-hub/awesome-substrate/blob/master/README.md    301!
+
+/vundefined                          /quick-start/                                 301!
+/vundefined/*                        /quick-start/                                 301!
+/concepts/off-chain-features/        /main-docs/fundamentals/offchain-operations/  301!
+/runtime/frame/                      /reference/frame-pallets/                     301!
+/tutorials/add-a-pallet/             /tutorials/work-with-pallets/add-a-pallet/    301!
+/runtime/upgrades/                   /main-docs/build/upgrade/                     301!
+/tutorials/forkless-upgrades/        /tutorials/get-started/forkless-upgrade/      301!
+/tools/                              /reference/command-line-tools/                301!
+/runtime/upgrades/                   /main-docs/build/upgrade/                     301!
+/getting-started/architecture/       /main-docs/fundamentals/architecture/         301!
```

Comment:
so to be clear @jonnysmillie - we need to _fix these links upstream_ not patch them over here long term, yes?

So to avoid confusion, can we be sure to clean this file up of redirects that we are reasonably sure of are no longer used anywhere?